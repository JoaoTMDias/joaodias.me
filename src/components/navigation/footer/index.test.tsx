import React from "react";
import { render } from "@testing-library/react";
import { Footer } from "./index";

describe("<Footer>", () => {
	it("should render without error", () => {
		const component = render(<Footer />);
		expect(component).toMatchSnapshot();
	});
});
