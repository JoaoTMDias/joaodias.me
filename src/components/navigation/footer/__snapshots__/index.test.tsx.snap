// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> should render without error 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="sc-AxgMl bwiHPM layout__container layout__section"
      >
        <div
          class="sc-AxjAm YToHs layout__container"
        >
          <div
            class="layout__row"
          >
            <div
              class="left"
            >
              <ul
                class="list"
              >
                <li
                  class="item"
                >
                  <a
                    aria-current="page"
                    class=""
                    href="/"
                  >
                    Homepage
                  </a>
                </li>
                <li
                  class="item"
                >
                  <a
                    href="/about/"
                  >
                    About
                  </a>
                </li>
                <li
                  class="item"
                >
                  <a
                    href="/contact/"
                  >
                    Contact
                  </a>
                </li>
              </ul>
            </div>
            <div
              class="right"
            >
              <ul
                class="list"
              >
                <li
                  class="item"
                >
                  <a
                    href="/privacy-policy/"
                  >
                    Privacy Policy
                  </a>
                </li>
                <li
                  class="item"
                >
                  <p>
                    2020 - Jo√£o Dias. All Rights Reserved
                  </p>
                </li>
              </ul>
            </div>
          </div>
        </div>
        <nav
          class="sc-AxirZ bJVfox"
          id="social-navigation"
          title="Social network links"
        >
          <ul
            class="sc-AxiKw ecNhrQ"
          >
            <li
              class="sc-AxhCb dciZQH"
            >
              <a
                aria-label="Follow me on Facebook"
                class="sc-AxhUy khqcPF"
                data-testid="component-external-link"
                href="https://fb.com/JoaoTMDias"
                rel="noopener noreferrer"
                tabindex="0"
                target="_blank"
                title="Follow me on Facebook"
              >
                <svg
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M12.88 22v-.07a10.55 10.55 0 0 1-1.13.07z"
                    fill="white"
                  />
                  <path
                    class="social__fill"
                    d="M15.12 13l1.13-2.81h-3.37V7.94A1.13 1.13 0 0 1 14 6.81h1.12V4h-2.24A3.38 3.38 0 0 0 9.5 7.38v2.81H7.25V13H9.5v8.74a10.41 10.41 0 0 0 2.25.26 10.55 10.55 0 0 0 1.13-.07V13z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                </svg>
              </a>
            </li>
            <li
              class="sc-AxhCb dciZQH"
            >
              <a
                aria-label="Follow me on Instagram"
                class="sc-AxhUy khqcPF"
                data-testid="component-external-link"
                href="https://instagram.com/JoaoTMDias"
                rel="noopener noreferrer"
                tabindex="0"
                target="_blank"
                title="Follow me on Instagram"
              >
                <svg
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="social__fill"
                    d="M15.75 2h-7.5A6.25 6.25 0 0 0 2 8.25v7.5A6.25 6.25 0 0 0 8.25 22h7.5A6.25 6.25 0 0 0 22 15.75v-7.5A6.25 6.25 0 0 0 15.75 2zm4.37 13.75a4.37 4.37 0 0 1-4.37 4.37h-7.5a4.37 4.37 0 0 1-4.37-4.37v-7.5a4.37 4.37 0 0 1 4.37-4.37h7.5a4.37 4.37 0 0 1 4.37 4.37z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                  <path
                    class="social__fill"
                    d="M12 7a5 5 0 1 0 5 5 5 5 0 0 0-5-5zm0 8.12A3.12 3.12 0 1 1 15.12 12 3.12 3.12 0 0 1 12 15.12z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                  <circle
                    cx="17.38"
                    cy="6.62"
                    fill="var(--color-gray5, #afb2be)"
                    r=".67"
                  />
                </svg>
              </a>
            </li>
            <li
              class="sc-AxhCb dciZQH"
            >
              <a
                aria-label="Check out my repos on Github"
                class="sc-AxhUy khqcPF"
                data-testid="component-external-link"
                href="https://github.com/Joaotmdias"
                rel="noopener noreferrer"
                tabindex="0"
                target="_blank"
                title="Check out my repos on Github"
              >
                <svg
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="social__fill"
                    d="M12 2.21a10 10 0 0 0-3.16 19.56c.5.1.68-.21.68-.48v-1.71c-2.78.61-3.37-1.34-3.37-1.34A2.64 2.64 0 0 0 5 16.77c-.91-.62.07-.61.07-.61a2.1 2.1 0 0 1 1.53 1 2.13 2.13 0 0 0 2.94.84 2.22 2.22 0 0 1 .63-1.35c-2.22-.25-4.55-1.11-4.55-5A3.89 3.89 0 0 1 6.65 9a3.62 3.62 0 0 1 .1-2.66s.84-.27 2.75 1a9.92 9.92 0 0 1 2.5-.28 10 10 0 0 1 2.51.34c1.9-1.3 2.74-1 2.74-1a3.56 3.56 0 0 1 .1 2.6 3.89 3.89 0 0 1 1 2.69c0 3.86-2.34 4.71-4.57 5a2.39 2.39 0 0 1 .68 1.86v2.75c0 .33.18.58.69.48A10 10 0 0 0 12 2.21z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                </svg>
              </a>
            </li>
            <li
              class="sc-AxhCb dciZQH"
            >
              <a
                aria-label="Check out some dribble shots"
                class="sc-AxhUy khqcPF"
                data-testid="component-external-link"
                href="https://dribbble.com/Joaotmdias"
                rel="noopener noreferrer"
                tabindex="0"
                target="_blank"
                title="Check out some dribble shots"
              >
                <svg
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="social__fill"
                    d="M12 2a10 10 0 1 0 10 10A10 10 0 0 0 12 2zm0 18.75A8.75 8.75 0 1 1 20.75 12 8.77 8.77 0 0 1 12 20.75z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                  <path
                    class="social__fill"
                    d="M16.79 6.23A7.47 7.47 0 0 0 12 4.5a7 7 0 0 0-1.53.16 36 36 0 0 1 2.46 4 18 18 0 0 0 3.86-2.43zM14.39 11.88a13.87 13.87 0 0 1 5.1-.09 7.44 7.44 0 0 0-1.81-4.68 19 19 0 0 1-4.21 2.68c.33.69.64 1.38.92 2.09zM9.21 5a7.48 7.48 0 0 0-4.51 5.31 17.6 17.6 0 0 0 7.06-1.16A34.25 34.25 0 0 0 9.21 5zM12.33 10.29a18.92 18.92 0 0 1-6.83 1.3h-1v.45a7.46 7.46 0 0 0 1.77 4.83 14.05 14.05 0 0 1 6.9-4.64c-.26-.68-.54-1.32-.84-1.94zM14.85 13.06a35.73 35.73 0 0 1 1.4 5.11A7.48 7.48 0 0 0 19.42 13a12.76 12.76 0 0 0-4.57.06zM7.16 17.72A7.43 7.43 0 0 0 12 19.5a7.52 7.52 0 0 0 3.11-.68 35.06 35.06 0 0 0-1.49-5.47 12.78 12.78 0 0 0-6.46 4.37z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                </svg>
              </a>
            </li>
            <li
              class="sc-AxhCb dciZQH"
            >
              <a
                aria-label="Follow me on Twitter"
                class="sc-AxhUy khqcPF"
                data-testid="component-external-link"
                href="https://twitter.com/Joaotmdias"
                rel="noopener noreferrer"
                tabindex="0"
                target="_blank"
                title="Follow me on Twitter"
              >
                <svg
                  fill="none"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="social__fill"
                    d="M24 4.309c-.892.391-1.843.65-2.835.776a4.893 4.893 0 0 0 2.165-2.719 9.833 9.833 0 0 1-3.12 1.191 4.919 4.919 0 0 0-8.511 3.365c0 .39.033.764.114 1.121-4.091-.199-7.71-2.16-10.142-5.146a4.954 4.954 0 0 0-.673 2.487c0 1.704.877 3.214 2.185 4.089A4.859 4.859 0 0 1 .96 8.867v.054a4.943 4.943 0 0 0 3.942 4.835c-.4.109-.837.162-1.29.162a4.35 4.35 0 0 1-.932-.084c.638 1.948 2.447 3.38 4.598 3.427a9.886 9.886 0 0 1-6.1 2.099c-.404 0-.791-.018-1.178-.068A13.852 13.852 0 0 0 7.548 21.5c9.054 0 14.004-7.5 14.004-14.001 0-.217-.008-.427-.018-.636A9.816 9.816 0 0 0 24 4.309z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                </svg>
              </a>
            </li>
          </ul>
        </nav>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="sc-AxgMl bwiHPM layout__container layout__section"
    >
      <div
        class="sc-AxjAm YToHs layout__container"
      >
        <div
          class="layout__row"
        >
          <div
            class="left"
          >
            <ul
              class="list"
            >
              <li
                class="item"
              >
                <a
                  aria-current="page"
                  class=""
                  href="/"
                >
                  Homepage
                </a>
              </li>
              <li
                class="item"
              >
                <a
                  href="/about/"
                >
                  About
                </a>
              </li>
              <li
                class="item"
              >
                <a
                  href="/contact/"
                >
                  Contact
                </a>
              </li>
            </ul>
          </div>
          <div
            class="right"
          >
            <ul
              class="list"
            >
              <li
                class="item"
              >
                <a
                  href="/privacy-policy/"
                >
                  Privacy Policy
                </a>
              </li>
              <li
                class="item"
              >
                <p>
                  2020 - Jo√£o Dias. All Rights Reserved
                </p>
              </li>
            </ul>
          </div>
        </div>
      </div>
      <nav
        class="sc-AxirZ bJVfox"
        id="social-navigation"
        title="Social network links"
      >
        <ul
          class="sc-AxiKw ecNhrQ"
        >
          <li
            class="sc-AxhCb dciZQH"
          >
            <a
              aria-label="Follow me on Facebook"
              class="sc-AxhUy khqcPF"
              data-testid="component-external-link"
              href="https://fb.com/JoaoTMDias"
              rel="noopener noreferrer"
              tabindex="0"
              target="_blank"
              title="Follow me on Facebook"
            >
              <svg
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12.88 22v-.07a10.55 10.55 0 0 1-1.13.07z"
                  fill="white"
                />
                <path
                  class="social__fill"
                  d="M15.12 13l1.13-2.81h-3.37V7.94A1.13 1.13 0 0 1 14 6.81h1.12V4h-2.24A3.38 3.38 0 0 0 9.5 7.38v2.81H7.25V13H9.5v8.74a10.41 10.41 0 0 0 2.25.26 10.55 10.55 0 0 0 1.13-.07V13z"
                  fill="var(--color-gray5, #afb2be)"
                />
              </svg>
            </a>
          </li>
          <li
            class="sc-AxhCb dciZQH"
          >
            <a
              aria-label="Follow me on Instagram"
              class="sc-AxhUy khqcPF"
              data-testid="component-external-link"
              href="https://instagram.com/JoaoTMDias"
              rel="noopener noreferrer"
              tabindex="0"
              target="_blank"
              title="Follow me on Instagram"
            >
              <svg
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  class="social__fill"
                  d="M15.75 2h-7.5A6.25 6.25 0 0 0 2 8.25v7.5A6.25 6.25 0 0 0 8.25 22h7.5A6.25 6.25 0 0 0 22 15.75v-7.5A6.25 6.25 0 0 0 15.75 2zm4.37 13.75a4.37 4.37 0 0 1-4.37 4.37h-7.5a4.37 4.37 0 0 1-4.37-4.37v-7.5a4.37 4.37 0 0 1 4.37-4.37h7.5a4.37 4.37 0 0 1 4.37 4.37z"
                  fill="var(--color-gray5, #afb2be)"
                />
                <path
                  class="social__fill"
                  d="M12 7a5 5 0 1 0 5 5 5 5 0 0 0-5-5zm0 8.12A3.12 3.12 0 1 1 15.12 12 3.12 3.12 0 0 1 12 15.12z"
                  fill="var(--color-gray5, #afb2be)"
                />
                <circle
                  cx="17.38"
                  cy="6.62"
                  fill="var(--color-gray5, #afb2be)"
                  r=".67"
                />
              </svg>
            </a>
          </li>
          <li
            class="sc-AxhCb dciZQH"
          >
            <a
              aria-label="Check out my repos on Github"
              class="sc-AxhUy khqcPF"
              data-testid="component-external-link"
              href="https://github.com/Joaotmdias"
              rel="noopener noreferrer"
              tabindex="0"
              target="_blank"
              title="Check out my repos on Github"
            >
              <svg
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  class="social__fill"
                  d="M12 2.21a10 10 0 0 0-3.16 19.56c.5.1.68-.21.68-.48v-1.71c-2.78.61-3.37-1.34-3.37-1.34A2.64 2.64 0 0 0 5 16.77c-.91-.62.07-.61.07-.61a2.1 2.1 0 0 1 1.53 1 2.13 2.13 0 0 0 2.94.84 2.22 2.22 0 0 1 .63-1.35c-2.22-.25-4.55-1.11-4.55-5A3.89 3.89 0 0 1 6.65 9a3.62 3.62 0 0 1 .1-2.66s.84-.27 2.75 1a9.92 9.92 0 0 1 2.5-.28 10 10 0 0 1 2.51.34c1.9-1.3 2.74-1 2.74-1a3.56 3.56 0 0 1 .1 2.6 3.89 3.89 0 0 1 1 2.69c0 3.86-2.34 4.71-4.57 5a2.39 2.39 0 0 1 .68 1.86v2.75c0 .33.18.58.69.48A10 10 0 0 0 12 2.21z"
                  fill="var(--color-gray5, #afb2be)"
                />
              </svg>
            </a>
          </li>
          <li
            class="sc-AxhCb dciZQH"
          >
            <a
              aria-label="Check out some dribble shots"
              class="sc-AxhUy khqcPF"
              data-testid="component-external-link"
              href="https://dribbble.com/Joaotmdias"
              rel="noopener noreferrer"
              tabindex="0"
              target="_blank"
              title="Check out some dribble shots"
            >
              <svg
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  class="social__fill"
                  d="M12 2a10 10 0 1 0 10 10A10 10 0 0 0 12 2zm0 18.75A8.75 8.75 0 1 1 20.75 12 8.77 8.77 0 0 1 12 20.75z"
                  fill="var(--color-gray5, #afb2be)"
                />
                <path
                  class="social__fill"
                  d="M16.79 6.23A7.47 7.47 0 0 0 12 4.5a7 7 0 0 0-1.53.16 36 36 0 0 1 2.46 4 18 18 0 0 0 3.86-2.43zM14.39 11.88a13.87 13.87 0 0 1 5.1-.09 7.44 7.44 0 0 0-1.81-4.68 19 19 0 0 1-4.21 2.68c.33.69.64 1.38.92 2.09zM9.21 5a7.48 7.48 0 0 0-4.51 5.31 17.6 17.6 0 0 0 7.06-1.16A34.25 34.25 0 0 0 9.21 5zM12.33 10.29a18.92 18.92 0 0 1-6.83 1.3h-1v.45a7.46 7.46 0 0 0 1.77 4.83 14.05 14.05 0 0 1 6.9-4.64c-.26-.68-.54-1.32-.84-1.94zM14.85 13.06a35.73 35.73 0 0 1 1.4 5.11A7.48 7.48 0 0 0 19.42 13a12.76 12.76 0 0 0-4.57.06zM7.16 17.72A7.43 7.43 0 0 0 12 19.5a7.52 7.52 0 0 0 3.11-.68 35.06 35.06 0 0 0-1.49-5.47 12.78 12.78 0 0 0-6.46 4.37z"
                  fill="var(--color-gray5, #afb2be)"
                />
              </svg>
            </a>
          </li>
          <li
            class="sc-AxhCb dciZQH"
          >
            <a
              aria-label="Follow me on Twitter"
              class="sc-AxhUy khqcPF"
              data-testid="component-external-link"
              href="https://twitter.com/Joaotmdias"
              rel="noopener noreferrer"
              tabindex="0"
              target="_blank"
              title="Follow me on Twitter"
            >
              <svg
                fill="none"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  class="social__fill"
                  d="M24 4.309c-.892.391-1.843.65-2.835.776a4.893 4.893 0 0 0 2.165-2.719 9.833 9.833 0 0 1-3.12 1.191 4.919 4.919 0 0 0-8.511 3.365c0 .39.033.764.114 1.121-4.091-.199-7.71-2.16-10.142-5.146a4.954 4.954 0 0 0-.673 2.487c0 1.704.877 3.214 2.185 4.089A4.859 4.859 0 0 1 .96 8.867v.054a4.943 4.943 0 0 0 3.942 4.835c-.4.109-.837.162-1.29.162a4.35 4.35 0 0 1-.932-.084c.638 1.948 2.447 3.38 4.598 3.427a9.886 9.886 0 0 1-6.1 2.099c-.404 0-.791-.018-1.178-.068A13.852 13.852 0 0 0 7.548 21.5c9.054 0 14.004-7.5 14.004-14.001 0-.217-.008-.427-.018-.636A9.816 9.816 0 0 0 24 4.309z"
                  fill="var(--color-gray5, #afb2be)"
                />
              </svg>
            </a>
          </li>
        </ul>
      </nav>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
